test/provider_os_memory_multiple_numa_nodes.cpp

```diff
@@ -88,13 +95,21 @@ struct testNuma : testing::Test {
         ASSERT_NE(os_memory_provider, nullptr);
     }
 
-    struct bitmask *retrieve_nodemask(void *addr) {
+    std::pair<int, bitmask *> retrieve_nodemask(void *addr) {
```

Comment:
why return via `pair`? can't we return bitmask via argument and just return `int` here...?

Reply:
The helper function is changed to void with IS_SKIPPED_OR_FAILED() in the main test